packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -89,7 +84,7 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
     onRenderAvatar
   } = props;
 
-  let localVideoTileStyles: VideoTileStylesProps = videoTileStyle;
+  let localVideoTileStyles: VideoTileStylesProps = {};
```

Comment:
If `shouldFloatLocalVideo` is false, this means that local video will be rendered with `{}` styles?

Reply:
These styles get recreated every render, could we put these in a useMemo?

Reply:
will put in useMemo.
Those styles are merged with default ones *(through mergeStyles), so even if empty it will work fine.
Also VideoTileStylesProps can not be of type null or undefined